test/plugin/scenarios/netty-http-4.1.x-scenario/support-version.list

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+4.1.25.Final
+4.1.51.Final
+4.1.91.Final
```

Comment:
Could you share why these versions? These are randomly picked or for some reasons?

Reply:
just randomly picked,should i reduce them?

Reply:
It is fine. I am just wondering whether there are some unsupported cases. Could you submit more to verify and exclude most in the final commits?

Reply:
> It is fine. I am just wondering whether there are some unsupported cases. Could you submit more to verify and exclude most in the final commits?

no problem

Reply:
> It is fine. I am just wondering whether there are some unsupported cases. Could you submit more to verify and exclude most in the final commits?

Should I reduce these test versions?

Reply:
No, the current status is good